src/routes/api/cryptor-migrations/migration-installation.ts

```diff
@@ -0,0 +1,65 @@
+import { Request, Response } from "express";
+import { Installation } from "models/installation";
+
+const MAX_BATCH_SIZE = 10_000;
+const DEFAULT_BATCH_SIZE = 10;
+const RESPONSE_BATCH_SIZE = 1000;
+
+export const CryptorMigrationInstallationPost = async (req: Request, res: Response): Promise<void> => {
+	const startTime = Date.now();
+
+	const batchSize = getValidBatchSize(req);
+
+	req.log = req.log.child({ operation: "migrate-installations-shared-secret", batchSize });
+
+	//--------- finding all in current batch to process --------------
+	req.log.info("About to start migrating installation sharedSecret");
+	const installations: Installation[] = await Installation.findAll({
+		limit: batchSize,
+		where: {
+			encryptedSharedSecret: null
+		}
+	});
+	if (installations.length === 0) {
+		req.log.info("No matching batch found, all done, returning now...");
+		res.status(200).send("No matching found, nothing to do");
+		return;
+	}
+	req.log.debug("Found matching installations to migrate, length " + installations.length);
+
+	//--------- loop and save each one of them --------------
+	let count = 0;
+	for (const inst of installations) {
+		//copy from shared secret to encryptedSharedSecret to enable encryption
+		inst.encryptedSharedSecret = inst.sharedSecret;
+		//update db
+		await inst.save();
+		req.log.info("Successfully migrated sharedSecret to encryptedSharedSecret");
```

Comment:
don't add this, going to _flood_ the production logs...

Reply:
Good point. fixed.
Do you think we should log the jiraHost when `inst.save()` throw an error?

Reply:
@mboudreau can you give advice on above?

Reply:
I mean, you're the one that provided the jiraHost...
I think you're fine.